core/chaincode/platforms/golang/hash.go

```diff
@@ -169,29 +169,21 @@ func getCodeFromHTTP(path string) (codegopath string, err error) {
 	case err = <-done:
 		// If we're here, the 'go get' command must have finished
 		if err != nil {
-			 err = fmt.Errorf("'go get' failed with error\n\"%s\"\n", err, string(errBuf.Bytes()))
+			err = fmt.Errorf("'go get' failed with error: \"%s\"\n%s", err, string(errBuf.Bytes()))
 		}
 	}
 	return
 }
 
 func getCodeFromFS(path string) (codegopath string, err error) {
 	logger.Debug("getCodeFromFS %s", path)
-	env := os.Environ()
-	var gopath string
-	for _, v := range env {
-		if strings.Index(v, "GOPATH=") == 0 {
-			p := strings.SplitAfter(v, "GOPATH=")
-			gopath = p[1]
-			break
-		}
-	}
-
```

Comment:
Ok, I think I found a happy medium. Reusing the old way of setting the env for the subcommand in a threadsafe manner while still ending up with a simpler piece of code.
And with the new commit, travis kicked in so that will also address any possible failure related to the comment I previously added with ci skip.